implementations/rust/ockam/ockam_command/src/tcp/outlet/list.rs

```diff
@@ -59,11 +61,21 @@ impl ListCommand {
 
         let (outlets, _) = try_join!(send_req, progress_output)?;
 
-        let list = opts.terminal.build_list(
-            &outlets.list,
-            &format!("Outlets on Node {}", node.node_name()),
-            &format!("No TCP Outlets found on node {}.", node.node_name()),
-        )?;
+        let list: String = {
+            let empty_message = fmt_info!(
+                "No TCP Outlets found on node {}",
+                color_primary(node.node_name())
+            );
```

Comment:
To enforce consistent formatting, we could pass a plain string to `build_list` for the `empty_message` argument and always build an `fmt_info` message with it.